backend/src/main/resources/application-prod.yml

```
@@ -24,3 +24,16 @@ app:
     expiration: 86400
   cors:
     allowed-origins: ${CORS_ALLOWED_ORIGINS}
+  mail:
+    enabled: ${MAIL_ENABLED:false}
+    from: ${MAIL_FROM:noreply@delivera.app}
+
+spring:
```

YAMLlint (1.38.0)

[error] duplication of key "spring" in mapping

(key-duplicates)